deployment/setup_20.04.sh

```diff
@@ -2,7 +2,7 @@
 
 # Description: Chatwoot installation script
 # OS: Ubuntu 20.04 LTS / Ubuntu 20.10
-# Script Version: 0.6
+# Script Version: 0.7
 # Run this script as root
 
 read -p 'Would you like to configure a domain and SSL for Chatwoot?(yes or no): ' configure_webserver
```

Comment:
should we show an additional help text here that, they need to have a domain with the A record pointing to the server Ip?

Reply:
@sojan-official We ask for a second confirmation and inform the user about the A-record requirement in the next step.

Reply:
https://github.com/chatwoot/chatwoot/blob/develop/deployment/setup_20.04.sh#L13